app/javascript/components/widgets/weather/weather-effect.jsx

```diff
@@ -460,9 +474,28 @@ const WeatherEffect = ({ weather }) => {
     },
   } = weather;
 
+  /* eslint-enable prettier/prettier */
   const Effect = weatherIdMap[id];
 
-  return Effect ? <EffectContainer>{Effect}</EffectContainer> : null;
+  const calcDuration = item => {
+    // eslint-disable-next-line no-shadow
+    const { Type, props, idx } = item;
+    if (idx > 53) {
```

Comment:
Is this to only apply the logic to Clouds? If that is the case it would make more sense to move the logic into `Cloud`.